### ISSUE TITLE
Select / Unselect does not work in Monthly view with userFormPopup: false

### ISSUE THREAD

**Issue by sabatale** [nhn/tui.calendar#1300]
## Version
Latest.

## Current Behavior
With `userFormPopup: true`, the select/unselect works fine. However, when set on false, the user cannot unselect the tile anymore. It just keeps adding color layers on top of each other. Or is because it's intended to be used in read-only when popups are set to false? You can observe this behaviour in the public demo as well.

[Example here](https://user-images.githubusercontent.com/50220345/194329777-896e6ffc-7cd4-4663-bd96-f2bdaaaf7508.gif)

## Expected Behavior
Select / Unselect should work the same across views.

**Comment by adhrinae** [comment]
@sabatale 

It's an intended behavior that selections can be accumulated.

In most cases, this can be easily solved by calling the [`clearGridSelection` method](https://github.com/nhn/tui.calendar/blob/main/docs/en/apis/calendar.md#cleargridselections).

But there's no option to clear a previous selection before selecting the new range automatically right now.

As a workaround, you can bind a `mousedown` event handler to the container element which calls the `clearGridSelection` method.

```js
document.getElementById('app').addEventListener('mousedown', () => {
  calendar.clearGridSelections();
});
```

---

Related(might be duplicated) to #1213.

**Comment by sabatale** [comment]
It makes sense to select multiple ranges, but I don't really see a use case for selecting the same item over and over.

In the previous versions, this behaviour was not there either. But yes, I understand the workaround, thanks for the tip!

**Comment by adhrinae** [comment]
@sabatale 

> It makes sense to select multiple ranges, but I don't really see a use case for selecting the same item over and over.

Yes, That's definitely a bug. But I missed the description you mentioned: "It just keeps adding color layers on top of each other."
